integrations/kml-parser/src/main/java/ch/stayhome/integrations/kml/job/ContactInformationGuesser.java

```diff
@@ -0,0 +1,67 @@
+package ch.stayhome.integrations.kml.job;
+
+import static org.apache.commons.lang3.StringUtils.trim;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.google.i18n.phonenumbers.PhoneNumberMatch;
+import com.google.i18n.phonenumbers.PhoneNumberUtil;
+import com.google.i18n.phonenumbers.Phonenumber;
+import org.apache.commons.lang3.StringUtils;
+
+
+public class ContactInformationGuesser {
```

Comment:
This class looks like a good candidate for a common lib to use in other integrations.